frontend/src/components/annotation/text_annotation/text_sidebar.vue

```diff
@@ -6,7 +6,7 @@
             top: ${toolbar_height}
         `"
     >
-        <v-expansion-panels multiple style="width: 350px;" accordion :value="open_panels">
+        <v-expansion-panels multiple style="width: 450px;" accordion :value="open_panels">
```

Comment:
Extended a width of the container that holds rendered text for better use of the real estate and to make it look a bit better.